docs/examples/ex18_square.py

```diff
@@ -0,0 +1,84 @@
+from skfem import *
+from skfem.models.poisson import vector_laplace, mass, laplace
+from skfem.models.general import divergence, rot
+
+import numpy as np
+from scipy.sparse import bmat
+
+from time import time
+
+mesh = MeshTri.init_tensor(*(np.linspace(0., 1., 2**5),)*2)
+
+element = {'u': ElementVectorH1(ElementTriP2()),
+           'p': ElementTriP1()}
+basis = {variable: InteriorBasis(mesh, e, intorder=3)
+         for variable, e in element.items()}
+
+
+@linear_form
+def body_force(v, dv, w):
+    return w.x[0] * v[1]
+
+tic = time()
+A = asm(vector_laplace, basis['u'])
+B = asm(divergence, basis['u'], basis['p'])
+C = asm(mass, basis['p'])
+
+K = bmat([[A, -B.T],
+          [-B, 1e-6 * C]]).tocsr()
+
+f = np.concatenate([asm(body_force, basis['u']),
+                    np.zeros(B.shape[0])])
+
+D = basis['u'].get_dofs().all()
+uvp = np.zeros(K.shape[0])
+uvp = solve(*condense(K, f, D=D))
+
+velocity, pressure = np.split(uvp, [A.shape[0]])
+print('elapsed time:', time() - tic)
+
+
+@linear_form
+def rot(v, dv, w):
```

Comment:
This is already imported and defined in skfem.models.general.

Reply:
Obsolete script, subsequently removed.